pkg/loki/loki.go

```diff
@@ -631,6 +631,7 @@ func (t *Loki) setupModuleManager() error {
 		Compactor:                {Server, Overrides, MemberlistKV, Analytics},
 		IndexGateway:             {Server, Store, IndexGatewayRing, IndexGatewayInterceptors, Analytics},
 		BloomGateway:             {Server, BloomGatewayRing, Analytics},
+		BloomCompactorRing:       {Server},
```

Comment:
It should have the same dependencies as other ring services:

```suggestion
		BloomCompactorRing:       {Overrides, MemberlistKV},
```